src/core_plugins/timelion/server/series_functions/holt/index.js

```diff
@@ -33,46 +34,64 @@ export default new Chainable('holt', {
     {
       name: 'alpha',
       types: ['number'],
-      help: `
+      help: i18n.translate('timelion.help.functions.holt.args.alphaHelpText', {
+        defaultMessage:
+          `
         Smoothing weight from 0 to 1.
         Increasing alpha will make the new series more closely follow the original.
-        Lowering it will make the series smoother`
+        Lowering it will make the series smoother`,
+      }),
     },
     {
       name: 'beta',
       types: ['number'],
-      help: `
+      help: i18n.translate('timelion.help.functions.holt.args.betaHelpText', {
+        defaultMessage:
+          `
         Trending weight from 0 to 1.
         Increasing beta will make rising/falling lines continue to rise/fall longer.
-        Lowering it will make the function learn the new trend faster`
+        Lowering it will make the function learn the new trend faster`,
+      }),
     },
     {
       name: 'gamma',
       types: ['number'],
-      help: `
+      help: i18n.translate('timelion.help.functions.holt.args.gammaHelpText', {
+        defaultMessage:
+          `
         Seasonal weight from 0 to 1. Does your data look like a wave?
         Increasing this will give recent seasons more importance, thus changing the wave form faster.
         Lowering it will reduce the importance of new seasons, making history more important.
-        `
+        `,
+      }),
     },
     {
       name: 'season',
       types: ['string'],
-      help: 'How long is the season, e.g., 1w if you pattern repeats weekly. (Only useful with gamma)'
+      help: i18n.translate('timelion.help.functions.holt.args.seasonHelpText', {
+        defaultMessage:
+          'How long is the season, e.g., 1w if you pattern repeats weekly. (Only useful with gamma)',
+      }),
     },
     {
       name: 'sample',
       types: ['number', 'null'],
-      help: `
+      help: i18n.translate('timelion.help.functions.holt.args.sampleHelpText', {
+        defaultMessage:
+          `
       The number of seasons to sample before starting to "predict" in a seasonal series.
-      (Only useful with gamma, Default: all)`
+      (Only useful with gamma, Default: all)`,
+      }),
```

Comment:
Add description: `"gamma" and "all" are parameter names and values and must not be translated.`